lib/container.ex

```diff
@@ -0,0 +1,115 @@
+defmodule RemoteDockers.Container do
+  alias RemoteDockers.{Client, HostConfig}
+  @moduledoc """
+  Connector to manage containers
+  """
+
+  @enforce_keys [:container_id, :host_config]
+  defstruct [:container_id, :host_config]
+
+  @containers_uri "/containers"
+
+  @doc """
+  List running containers
+  """
+  def list!(host_config) do
```

Comment:
Shouldn't we force these functions arguments types?

Reply:
We should test typespec to improve that.